packages/py-api/py_api/controllers/teams_controller.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from bson.errors import InvalidId
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
+
+
+class TeamsController:
+
+    def fetch_teams() -> JSONResponse:
+        teams = list(t_col.find())
+
+        if not teams:
+            return JSONResponse(content={"message": "No teams were found in db"}, status_code=404)
+
+        return JSONResponse(content={"teams": json.loads(dumps(teams))}, status_code=201)
+
+    def get_team(object_id: str) -> JSONResponse:
+        try:
+            specified_team = t_col.find_one(
+                filter={"_id": ObjectId(object_id)},
+            )
+        except (InvalidId, TypeError):
+            return JSONResponse(content={"message": "Invalid object_id format!"}, status_code=400)
```

Comment:
Can we extract the exception handler for these errors as a middleware, so that we stick to the DRY principle?